kb/temporal-platform-limits-sheet.md

```diff
@@ -9,38 +9,38 @@ date: 2023-04-28T00:00:00Z
 ---
 
 Running into limits can cause unexpected failures.
-Knowing Temporal's limits can prevent that.
+Knowing the limits of Temporal can prevent that.
 
-This page details many of the hard and soft limits that are coded into the Temporal Platform.
-Hard limits fail with an error; soft limits produce a warning log on the server side.
+This page details many of the hard and soft limits coded into the Temporal Platform.
+Hard limits fail with an error. Soft limits produce a warning log on the server side.
 
 <!-- truncate -->
 
-- **Identifiers**: By default, the maximum length for identifiers (like Workflow Id, Workflow Type, and Task Queue name) is 1000 characters.
-  - This is configurable with the `limit.maxIDLength` dynamic config, which is set to 255 in [our SQL example](https://github.com/temporalio/docker-compose/blob/93d382ef9133e4cde8ce311de5153cd0cc9fbd0c/dynamicconfig/development-sql.yaml#L1-L2).
+- **Identifiers**: By default, the maximum length for identifiers (such as Workflow Id, Workflow Type, and Task Queue name) is 1000 characters.
```

Comment:
Merely FYI: Google style is OK with using "like" for examples. (My personal style, FWIW, is to use "such as", so I have no issue with this change.)